modules/virt-add-disk-to-vm.adoc

```diff
@@ -24,22 +26,34 @@ Use this procedure to add a virtual disk to a {object}.
 
 .Procedure
 
-. From the *{object-gui}* tab, select your {object}.
-. Select the *Disks* tab.
-. Click *Add Disks* to open the *Add Disk* window.
-. In the *Add Disk* window, specify *Source*, *Name*, *Size*, *Interface*, and *Storage Class*.
+. Click *Workloads* -> *Virtualization* from the side menu.
+. Click the *{object-gui}s* tab.
+. Select a {object} to open the *{object-gui} Overview* screen.
+. Click the *Disks* tab.
+. Click *Add Disk* to open the *Add Disk* window.
+. In the *Add Disk* window, specify *Source*, *Name*, *Size*, *Interface*, *Type*, and *Storage Class*.
 .. Optional: In the *Advanced* list, specify the *Volume Mode* and *Access Mode* for the virtual disk. If you do not specify these parameters, the system uses the default values from the `kubevirt-storage-class-defaults` ConfigMap.
-. Use the drop-down lists and check boxes to edit the disk configuration.
-. Click *OK*.
+. Click *Add*.
+
+ifdef::virt-vm[]
+[NOTE]
+====
+If the {object} is running, the new disk is in the *pending restart* state and will not be attached until you restart the {object}.
+
+A list of all pending changes, that will be applied when the {object} restarts, is displayed in the *Pending Changes* banner on top of the page.
```

Comment:
This sentence needs a couple of tweaks (passive voice, commas, + wording) so I have a rewrite suggestion for you to take or leave: 

`The *Pending Changes* banner at the top of the page displays a list of all changes that will be applied when the {object} restarts.`

(Applies to all instances of this sentence in the PR)

Reply:
Changes applied to all instances of the sentence